04-Honduras/02-R/1-import/00_Master.R

```diff
@@ -0,0 +1,135 @@
+# ------------------------------------------------------------------------------ #
+#                          ONCAE - Honduras Procurement Project                  #
+#                                       World Bank                               #
+#                                       Nov - 2022                               #
+#                                                                                #
+# ------------------------------------------------------------------------------ #
+# --------------------------------------- #
+#     Authors: Hao Lyu (RA - DIME)  #                                       
+# --------------------------------------- #
+
+# ****************************************************************************** #
+#   This is the master code: running this script will allow you to run all the   #
+#   other scripts                                                                #
+#                                                                                #
+# ****************************************************************************** #
+# 11/10/2022
+
+######################
+####   SETUP    ######
+######################
+
+#### Clear the environment -----------------------------------------------------
+
+rm(list=ls())
+
+#### Set the current working directory -----------------------------------------
+
+if (Sys.info()["user"] == "haolyu") {
+  
+  scriptFolder <- file.path("/Users/haolyu/Documents/GitHub/honduras-procurement") 
+  
+  projectFolder <- file.path("/Users/haolyu/Dropbox/Honduras_Procurement") 
+
+} else if (Sys.info()["user"] == ""){
+  
+  scriptFolder <- file.path("")
+  
+  projectFolder <- file.path("")
+  
+} # <- put your path here 
+
+scripts     <- file.path(scriptFolder,"Scripts/Master"                                             ) # folder for the scripts
+outputs     <- file.path(projectFolder,"Outputs"                                                    ) # folder for all outputs produced
+raw_Data    <- file.path(projectFolder,"Data/Raw"                                                   ) # folder for all raw datasets
+Data        <- file.path(projectFolder,"Data/DCC/Intermediate/1 - Panel data"                       )
+Data_final  <- file.path(projectFolder,"Data/DCC/Intermediate/2 - Harmonized sub-sampled Panel Data")
+final       <- file.path(projectFolder,"Data/DCC/Final")
+data        <- file.path(projectFolder,"Data")
+
```

Comment:
Hao, I thought you were using only intermediate data (cleaned data from Ruggero), right? If yes, I think there are paths to raw data. Can you add a note when you use intermediate and when raw data? Thank you!